tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -665,5 +724,40 @@ public sealed override int ReturnSomething()
                 return 10;
             }
         }
+
+        public class FooFactory : DummyFactory<IFoo>, IFoo

```

Comment:
The fact that the class is both an implementation of `IFoo` and a `DummyFactory<IFoo>` is a bit confusing... Make them separate perhaps?